nodestream_akamai/plugin.py

```diff
@@ -0,0 +1,17 @@
+from nodestream.project import Project, ProjectPlugin, PipelineScope
+
+class AkamaiPlugin(ProjectPlugin):
+    def activate(self, project: Project) -> None:
+        scopes = [
+            PipelineScope.from_resources(name="appsec-coverage", package="nodestream_akamai.appsec_coverage"),
+            PipelineScope.from_resources(name="cps", package="nodestream_akamai.cps"),
+            PipelineScope.from_resources(name="ehn", package="nodestream_akamai.ehn"),
+            PipelineScope.from_resources(name="gtm", package="nodestream_akamai.gtm"),
+            PipelineScope.from_resources(name="netstorage", package="nodestream_akamai.netstorage"),
+            PipelineScope.from_resources(name="propery", package="nodestream_akamai.property"),
+            PipelineScope.from_resources(name="redirect", package="nodestream_akamai.redirect"),
+            PipelineScope.from_resources(name="siteshield", package="nodestream_akamai.siteshield"),
+            PipelineScope.from_resources(name="waf", package="nodestream_akamai.waf"),
```

Comment:
For the scope names, should we add a `akamai_` prefix to them? What if a person had a `waf` scope for something else, it would collide. e.g `akamai_cps`.